modules/consensus/setdb.go

```diff
@@ -0,0 +1,226 @@
+package consensus
+
+import (
+	"errors"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/encoding"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/persist"
+	"github.com/NebulousLabs/Sia/types"
+	"github.com/boltdb/bolt"
+)
+
+var meta = persist.Metadata{
+	Version: "0.1",
+	Header:  "Consensus Set Backing Database",
+}
+
+var (
+	errBadSetInsert = errors.New("attempting to add an already existing item to the consensus set")
+	errNilBucket    = errors.New("using a bucket that does not exist")
+	errNilItem      = errors.New("Requested item does not exist")
+)
+
+// setDB is a wrapper around the persist bolt db which backs the
+// consensus set
+type setDB struct {
+	*persist.BoltDatabase
+}
```

Comment:
The wrapper lets us have consensus specific member functions. Also there will eventually be additional items in the struct like error flags.